internal/grpc/services/storageprovider/storageprovider.go

```diff
@@ -739,14 +801,17 @@ func (s *service) trimMountPrefix(fn string) (string, error) {
 	return "", errors.New(fmt.Sprintf("path=%q does not belong to this storage provider mount path=%q"+fn, s.mountPath))
 }
 
-func getFS(c *config) (storage.FS, error) {
-	if f, ok := registry.NewFuncs[c.Driver]; ok {
-		return f(c.Drivers[c.Driver])
+func (s *service) wrap(ctx context.Context, ri *storageproviderv0alphapb.ResourceInfo) error {
+	ri.Id.StorageId = s.mountID
```

Comment:
We must not change storage ids

Reply:
That is the code from the old wrap(). AFAICT it is used to fill the storage id with the storage providers mountid, because the storage driver does not provide it.